acm-repos/kf-ci-v1/namespaces/auto-deploy/apps_v1_statefulset_test-pod.yaml

```diff
@@ -19,7 +19,7 @@ spec:
         - tail
         - -f
         - /dev/null
-        image: gcr.io/kubeflow-ci/test-worker-py3:d67009b@sha256:e96d069db00159c03b95ed1064569bcd0a78acdc691f42b5025050370949007a
+        image: gcr.io/kubeflow-ci/test-worker-py3:b23b63b-dirty@sha256:a749d7fa4d77466c892a206d3adf0909e86717da898dbd12378e6cbed59ffbd3
```

Comment:
How come this image is changing? Did you build a new image?

Reply:
NM; I'm guessing we hadn't rehydrated the manifests since a newer image had been built.

Reply:
Do you think it can break something just in case? If not I would like to use tag [206de74](https://console.cloud.google.com/gcr/images/kubeflow-ci/GLOBAL/test-worker-py3@sha256:a749d7fa4d77466c892a206d3adf0909e86717da898dbd12378e6cbed59ffbd3/details?tab=info) instead.